### ISSUE TITLE
Build failure: build/external/coffee-script_1.10.0/bin/coffee: No such file or directory

### ISSUE THREAD

**Issue by mpaladin** [rethinkdb/rethinkdb#6362]
Build failure since https://github.com/rethinkdb/rethinkdb/commit/3710301531a78d6dd867d14fe7631a6b6b3c1f26. Reproducer on a clean ubuntu linux:
```
./configure --allow-fetch
make
```
Error:
```
    [166/711] FETCH node_6.10.0
    [167/711] BUILD node_6.10.0
    [168/711] FETCH coffee-script_1.10.0
    [169/711] BUILD coffee-script_1.10.0
    [170/711] COFFEE build/drivers/javascript/coffee/cursor.js
/bin/bash: /tmp/rethinkdb/build/external/coffee-script_1.10.0/bin/coffee: No such file or directory
drivers/javascript/build.mk:25: recipe for target 'build/drivers/javascript/coffee/cursor.js' failed
make[1]: *** [build/drivers/javascript/coffee/cursor.js] Error 127
make[1]: *** Deleting file 'build/drivers/javascript/coffee/cursor.js'
Makefile:52: recipe for target 'make' failed
make: *** [make] Error 2
```

**Comment by AtnNn** [comment]
@srh @mpaladin I found the cause of this issue.

To fix #6356, I created a wrapped for `npm` and moved the original `npm` to `npm.orig`.

Using `npm.sucks` instead of `npm.orig` seems to fix this issue. I wonder why?

```
  // if npm is called as "npmg" or "npm_g", then
  // run in global mode.
  if (path.basename(process.argv[1]).slice(-1) === 'g') {
    process.argv.splice(1, 1, 'npm', '-g')
  }
```

**Comment by srh** [comment]
Closing -- the changes that broke things have been reverted.
